packages/core/src/__tests__/state-manager.test.ts

```diff
@@ -17,12 +17,32 @@ import { TileDocument } from '@ceramicnetwork/stream-tile'
 import { streamFromState } from '../state-management/stream-from-state.js'
 import * as uint8arrays from 'uint8arrays'
 import * as sha256 from '@stablelib/sha256'
-import { StreamID, CommitID } from '@ceramicnetwork/streamid'
+import { CommitID, StreamID } from '@ceramicnetwork/streamid'
 import { from, timer } from 'rxjs'
-import { concatMap, map } from 'rxjs/operators'
+import { concatMap, map, tap } from 'rxjs/operators'
 import { MAX_RESPONSE_INTERVAL } from '../pubsub/message-bus.js'
 import cloneDeep from 'lodash.clonedeep'
 import { StateLink } from '../state-management/state-link.js'
+import { InMemoryAnchorService } from '../anchor/memory/in-memory-anchor-service.js'
+
+const waitForAnchorStatus = async (stream$: RunningState, status: AnchorStatus): Promise<void> => {
+  await new Promise<void>((resolve) => {
+    let resolved = false
+    stream$.subscribe()
```

Comment:
why is that?

Reply:
Thanks! I missed that left-over. It's removed now.